lms/templates/admin/base.html.jinja2

```diff
@@ -45,7 +53,7 @@
   </nav>
   {% endblock %}
   <div class="block has-text-centered mt-5">
-      <h1 class="title">{% block header %} Admin {% endblock %}</h1>
+      <h1 class="title">{% block header %}{% endblock %}</h1>
```

Comment:
This play nicely with not having a header and getting a placeholder title.

This gets rendered as a big `<h1>` on the page so the default "**ADMIN**" doesn't seem that useful anyway.